src/index.html

```diff
@@ -19,9 +19,388 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
The `href` attribute should point to a `.css` file, not a `.scss` file. Make sure to compile your SCSS to CSS and link to the compiled CSS file.